components/header.js

```diff
@@ -30,17 +24,14 @@ export const Header = ({ menu }) => {
             width: 100vw;
             z-index: 100;
           }
+
           .navbar {
             display: flex;
             height: 80px;
             align-items: center;
             justify-content: space-between;
           }
-          .menu {
-            display: flex;
-            justify-content: center;
-            width: 900px;
-          }
+
           .header :global(a) {
```

Comment:
I think since you created a Menu component it would be better to move all "a" styles there